src/wpilib/robot-states.md

```diff
@@ -0,0 +1,36 @@
+# Robot States
+
+Robots programmed with WPILib can be in one of four states:
```

Comment:
What is a state? Maybe: "States are different operating modes of the robot that control how it behaves"